app/models/kudo.rb

```diff
@@ -9,15 +9,15 @@ class Kudo < ApplicationRecord
   validate :cannot_be_author
   validate :guest_cannot_kudos_restricted_work
 
-  validates_uniqueness_of :pseud_id,
-    scope: [:commentable_id, :commentable_type],
-    message: ts("^You have already left kudos here. :)"),
-    if: Proc.new { |kudo| !kudo.pseud.nil? }
-
   validates_uniqueness_of :ip_address,
-    scope: [:commentable_id, :commentable_type],
-    message: ts("^You have already left kudos here. :)"),
-    if: Proc.new { |kudo| !kudo.ip_address.blank? }
+                          scope: [:commentable_id, :commentable_type],
+                          message: ts("^You have already left kudos here. :)"),
```

Comment:
The carets did work once upon a time:

https://github.com/otwcode/otwarchive/blob/2352531b7e024ac77ecd29ceb832620752fc3646/app/controllers/kudos_controller.rb#L12

To be cleaned up in [AO3-5635](https://otwarchive.atlassian.net/browse/AO3-5635).